### PR TITLE
[backport] Use default dtype in `L.NegativeSampling`

### PR DESCRIPTION
**Pull request by takagi**
Backport of #5664

### CODE REVIEW

**Comment by kmaehashi** [comment]
Jenkins, test this please.

File: chainer/links/loss/negative_sampling.py [hunk 1]
```diff
@@ -28,12 +29,13 @@ class NegativeSampling(link.Link):
 
     """
 
-    def __init__(self, in_size, counts, sample_size, power=0.75):
+    def __init__(self, in_size, counts, sample_size, power=0.75, dtype=None):
         super(NegativeSampling, self).__init__()
+        dtype = chainer.get_dtype(dtype)
```
Comment (toslunar) [inline chainer/links/loss/negative_sampling.py]: import `chainer`.

Comment (takagi) [reply chainer/links/loss/negative_sampling.py]: Fixed! Missing `import chainer` line looks imported to the master in #5725, ChainerX integration.

**Comment by takagi** [comment]
Jenkins, test this please.

**Pull request merged**